.github/workflows/pr.yml

```diff
@@ -5,14 +5,14 @@ on:
       - main
 
 jobs:
-  dependencies:
-    uses: the-guild-org/shared-config/.github/workflows/changesets-dependencies.yaml@main
-    with:
-      installDependencies: true
-      packageManager: yarn
-      nodeVersion: 18
-    secrets:
-      githubToken: ${{ secrets.GUILD_BOT_TOKEN }}
+#   dependencies:
```

Comment:
:)))

Reply:
temporary until @dotansimha resolves it 😃